src/_cffi_src/openssl/x509.py

```diff
@@ -65,6 +65,7 @@
 typedef struct {
     X509_CRL_INFO *crl;
     X509_ALGOR *sig_alg;
+    ASN1_BIT_STRING *signature;
```

Comment:
Yeah, OpenSSL 1.1.0 is opaquing almost every struct. In most cases there are already accessors available, but for this (and the tbs payload) we'll have to define some functions since cffi won't know what the struct looks like.